jboss-kie-smartrouter/tests/bats/jboss-kie-smartrouter-common.bash

```diff
@@ -0,0 +1,9 @@
+load ../../../tests/bats/common/logging
```

Comment:
no need to create a new file just to hold only one common method. I think we can remove it, and as said on the next comment, use diff instead it.

Reply:
ok

Reply:
ok